assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -176,7 +172,7 @@ const AlertsList: ComponentType<AlertsListProps> = ({
 
       return screensWithAlert
         ? screensWithAlert.find((screen_id) =>
-            ids.includes(screenMetaData[screen_id].type)
+            ids.includes(screenMetaData[screen_id]?.type)
```

Comment:
Was this in the original diff? Wondering why the `?` needed to be added.

Reply:
So I think this may have been a bandaid for the `screens_by_alert` bug that was fixed recently. Screens that were hidden from Screenplay were still logging alerts to the cache. When filtering by screen type, Screenplay would crash because of a null reference. Now that we aren't caching hidden screens, this is no longer necessary.